stanford_profile.post_update.php

```diff
@@ -46,3 +49,58 @@ function stanford_profile_post_update_8014() {
   ]);
 
 }
+
+/**
+ * Create blocks for sites with custom themes that were added to stanford_basic.
+ */
+function stanford_profile_post_update_8015() {
+  $theme_name = \Drupal::config('system.theme')->get('default');
+  // Default theme is good. Just end if so.
+  if ($theme_name == "stanford_basic") {
+    return;
+  }
+
+  // Not stanford_basic, we have to create two config page blocks.
+  // Copy the blocks from stanford_basic and rename them.
+  //
+  // Names of things.
+  $basic_global_name = 'block.block.stanford_basic_config_pages_stanford_global_msg';
+  $basic_super_name = 'block.block.stanford_basic_config_pages_stanford_super_footer';
+  $my_global_name = 'block.block.' . $theme_name . '_config_pages_stanford_global_msg';
+  $my_super_name = 'block.block.' . $theme_name . '_config_pages_stanford_super_footer';
+
+  // Resources.
+  $config_path = Settings::get('config_sync_directory');
+  $source = new FileStorage($config_path);
+  $config_storage = \Drupal::service('config.storage');
```

Comment:
this doe the same thing. i'd suggest using the `$config_storage`. Its currently unused also

Reply:
Gone

Reply:
/giphy bye bye bye

[image of backstreet boys]

Reply:
I would've preferred to use the service. but :man_shrugging:

Reply:
Isn't it technically possible to change config.storage to something other than the file storage?